src/util/time.h

```diff
@@ -24,8 +24,15 @@ void UninterruptibleSleep(const std::chrono::microseconds& n);
  * This helper is used to convert durations before passing them over an
  * interface that doesn't support std::chrono (e.g. RPC, debug log, or the GUI)
  */
-inline int64_t count_seconds(std::chrono::seconds t) { return t.count(); }
-inline int64_t count_microseconds(std::chrono::microseconds t) { return t.count(); }
+constexpr inline int64_t count_seconds(std::chrono::seconds t) { return t.count(); }
+constexpr inline int64_t count_microseconds(std::chrono::microseconds t) { return t.count(); }
```

Comment:
```suggestion
constexpr int64_t count_microseconds(std::chrono::microseconds t) { return t.count(); }
```

nit: constexpr implies inline, so no need to specify twice

Reply:
Done.